src/abstractdataframe/abstractdataframe.jl

```diff
@@ -873,9 +873,9 @@ Return a copy of data frame `df` containing only rows for which `function`
 returns `true`.
 
 If `cols` is not specified then the function is passed `DataFrameRow`s.
-If `cols` is a single column then then elements of column `cols` are passed to the function.
-If `cols` represents multiple columns (any valid column selector is accepted) then
-`NamedTuple`s formed with elements of the selected columns are passed to the function.
+If `cols` argument is passed then it should be valid column selector (additionally
+allowing column duplicates are allowed if a vector of `Int` or `Symbol is passed),
+the the function is passed elements of the selected columns as consecutive positional arguments.
```

Comment:
```suggestion
the function is passed elements of the selected columns as separate positional arguments.
```
(I'm not sure "consecutive" adds anything here, we can't pass non-consecutive arguments. :-)